build/CoreGraphics/lib/CoreGraphicsLib.vcxproj

```diff
@@ -54,10 +54,10 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFStream.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFString.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGShading.mm" />
-    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\D2DWrapper.mm" />
+    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\D2DUtility.mm" />
```

Comment:
Given it had WIC methods, I thought it might be sane., but Wrapper seems more generic.

---
In reply to: [84149288](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84149288) [](ancestors = 84149288)